browser/resources/brave_new_tab_page_refresh/stories/sb_locale.ts

```diff
@@ -96,13 +160,18 @@ const localeStrings: { [K in StringKey]: string }  = {
   widgetSettingsTitle: 'Widgets'
 }
 
+const pluralStrings: { [K in PluralStringKey]: string }  = {
+  newsSourceCountText: '# sources'
+}
+
 export function createLocale() {
-  const getString =
-    (key: string) => String((localeStrings as any)[key] || 'MISSING')
   return {
-    getString,
+    getString(key: string) {
+      return String((localeStrings as any)[key] || 'MISSING')
```

Comment:
Could we change the argument from `type: string` to `type: StringKey` so the sites with pass something which isn't a valid key get a build error?

Reply:
Yes - the returned object should not be more general than the `Locale` interface it's supposed to return. Thanks for spotting!